src/main/java/tools/jackson/databind/deser/bean/BeanDeserializerBase.java

```diff
@@ -1711,7 +1711,9 @@ protected void handleUnknownProperty(JsonParser p, DeserializationContext ctxt,
             Object beanOrClass, String propName)
         throws JacksonException
     {
-        if (_ignoreAllUnknown) {
```

Comment:
Make sure you know how and why `_ignoreAllUnknown` is set to `true`.

For example, `@JsonIgnoreProperties(ignoreUnknown = true)` on Class must have precedence over global `FAIL_ON_UNKNOWN_PROPERTIES`.

And I think this change breaks that: `BeanDeserializerFactory` line 525 is where this setting is passed.